tests/Doctrine/DBAL/Migrations/Tests/Finder/RecursiveRegexFinderTest.php

```diff
@@ -63,6 +63,7 @@ public function testFindMigrationsReturnsTheExpectedFilesFromDirectory()
             '20150502000004' => 'TestMigrations\\Version20150502000004',
             '20150502000005' => 'TestMigrations\\Version20150502000005',
             '1_reset_versions' => 'TestMigrations\\Version1_reset_versions',
+            '1_symlinked_file' => 'TestMigrations\\Version1_symlinked_file',
```

Comment:
That function is going to be exactly the same, and the test name is `testFindMigrationsReturnsTheExpectedFilesFromDirectory` which quite well describes this new test as well.

What about calling this test two times, with different dirs (using dataProvider) and from the second directory I would symlink the original directory?